doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
I don't think the documentation is wrong, or at least not completely wrong. As would be expected from the docs, this does nothing: `process.kill(process.pid, 'SIGPIPE');`

Perhaps the `EPIPE` errors that epipebomb suppresses aren't coming from `SIGPIPE` signals (or at least not directly from `SIGPIPE` signals) but instead are just straight up errors from function calls.